### ISSUE TITLE
`bal.bat update` fails on win-64 because of somehow misparsing the `install.bat` file.

### ISSUE THREAD

**Issue by slippycheeze** [ballerina-platform/ballerina-update-tool#177]
**Description:**
`bal.bat update` results in a string of errors that, clearly, are triggered by something going wrong with windows understanding the command – `install.bat` from the new download, I believe – to be run to emplace the new version of the update tool.

I'm not sure the problem doesn't originate earlier, though, since the start of the problem is:
```
C:\Users\danie>bal update
Fetching the latest update tool version from the remote server...
Downloading ballerina-command-1.3.5 100% [============================================================] 1/1 MB (0:00:00 / 0:00:00)
'--------------------------------------------------------------------' is not recognized as an internal or external command,
operable program or batch file.
"2019," was unexpected at this time.
'2' is not recognized as an internal or external command, operable program or batch file.
```

Given that the bundled update tool with the swan lake beta2 install is 1.3.6, it seems like the attempt to downgrade is probably also a bug.

Full output from my system: [bal-update-fail.txt](https://github.com/ballerina-platform/ballerina-update-tool/files/7129182/bal-update-fail.txt)

**Steps to reproduce:**
1. Download swan-lake-beta2 from ballerina.io
2. unpack anywhere; this happened with both the MSI installer in the default location and the zip install in `~/opt/ballerina`
3. restart terminal emulator to get the new PATH in place, or whatever floats your boat there.
4. run `bal update` or `bal.bat` update
   - reproduces for me with powershell 7, and with cmd.exe, so isn't dependent on the initial command processor
5. watch the system not cope. 
6. realise that it did successfully delete the `ballerina-command` jar, but didn't put the new one in, so broke the install as well. :)

**Affected Versions:**
swan-lake-beta2, which has:
```
PS C:\Users\danie\opt\ballerina> bal version
Ballerina Swan Lake Beta 2
Language specification 2021R1
Update Tool 1.3.6
```

**OS, DB, other environment details and versions:**
Windows 10 pro, 64-bit, Version `21H1 (OS Build 19043.1165)`

**Comment by keizer619** [comment]
There was an issue `bal update` command which downloads 1.3.5 for 1.3.6. We have fixed it now. We will check whether is it still happening in 1.3.6.

**Comment by NipunaMadhushan** [comment]
After the fix, the issue is not recreating on windows.

**Comment by keizer619** [comment]
@slippycheeze 
Since this is fixed along with 1.3.6 we will be closing this issue. If there are any concerns please feel free to reopen :) 
thanks for reporting
